config/webpack/webpack.client.babel.ts

```diff
@@ -185,7 +188,7 @@ export default {
           'delay',
           'immer',
           'immer',
-          'lodash',
+          // 'lodash',
```

Comment:
`lodash` is used by the algorithm, and the algorithm is imported into a webworker. If this line left uncommented, lodash is modified by babel for hot reloading, which results in webworker throwing an error. This fix needs a better implementation.

Reply:
@abrie This stuff you are pointing to is only needed in prod, for better tree shaking and in case the package does not ship an es5 version, in order to support old browsers. But hot reloading is dev only, so I think we can make it work by conditioning additional transpilation to prod only. Should also make dev builds a bit faster.

Reply:
@ivan-aksamentov is that a task for a new PR? It's not immediately obvious to me how to add a production-only conditional in there... need some help. :)

Reply:
@abrie I added this to master in  #600, please rebase/merge.

Reply:
@ivan-aksamentov works great!